e/eventsource/eventsource_rhel_ubi_8.sh

```diff
@@ -16,7 +17,7 @@
 # ----------------------------------------------------------------------------
 
 PACKAGE_NAME=eventsource
-PACKAGE_VERSION=0.1.6
+PACKAGE_VERSION=v0.0.1
 PACKAGE_URL=https://github.com/EventSource/eventsource
 
 yum -y update && yum install -y yum-utils nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git gcc gcc-c++ libffi libffi-devel ncurses git jq make cmake
```

Comment:
Do we need all these packages to be installed ?

It was there in earlier script, because, it was generated by Automation.
Can you please check and remove unwanted package installation ?

Reply:
Hi @kandarpamalipeddi removed unwanted packages